astroquery/conftest.py

```diff
@@ -50,8 +50,7 @@
     pass
 
 # ignoring pyvo can be removed once we require >0.9.3
-enable_deprecations_as_exceptions(include_astropy_deprecations=False,
```

Comment:
I'm afraid we should keep this, unless we subclass the warning and use our own. Which would make sense, but maybe separately.

Also we can require pyvo 1.0, in fact the `cadc` module requires that version.

Reply:
I reverted the change here. 

This is weird because I was simply reverting #1517, which was necessary because deprecation over at `astropy` because I moved the timer stuff here. New deprecation warnings unrelated to this work must have crept in since then.

`pyvo` pinning should probably be a separate PR.

Reply:
yes, I added this deprecation ~2-3 PRs ago to get rid of the FutureWarnings :)